plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/SettingsForm.js

```diff
@@ -95,6 +103,22 @@ Ext.define('NX.controller.SettingsForm', {
     }
   },
 
+  /*
+   * @private
+   * Reset the dirty state of the form
+   */
+  resetDirtyState: function (form) {
```

Comment:
This looks like it's doing an end run around the available API, and a google search suggests we may be able to do this more easily (if it's recent enough for our version of ExtJS of course :) )
http://snehalvetal.blogspot.ca/2012/08/extjs-clearing-dirty-flags-for-form.html